test/mobile/features/android/android_ndk_errors.feature

```diff
@@ -6,7 +6,9 @@ Feature: Android manual smoke tests
     @skip_unity_android_2020
     Scenario: NDK Signal raised
         When I tap the "NDK signal" button
+        And I wait for 8 seconds
```

Comment:
This probably won't be needed post- #302 ?

Reply:
it's needed be cause there is a delay of 6 seconds after pressing the NDK signal button and the signal happening

Reply:
i merged our tests and kept the wait logic and ran the tests to make sure it's good

Reply:
Ah got it - I'd add a note to the effect of saying that the signal is fired after the app is fully launched in a few seconds or something